src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -3475,9 +3475,26 @@ namespace InternalHeatGains {
                         thisZoneBBHeat.EndUseSubcategory = "General";
                     }
 
-                    thisZoneBBHeat.CapatLowTemperature = IHGNumbers(1);
+                    Real64 spaceFrac = 1.0;
+                    if (thisBBHeatInput.numOfSpaces > 1) {
+                        Real64 const zoneArea = state.dataHeatBal->Zone(zoneNum).FloorArea;
+                        if (zoneArea > 0.0) {
+                            spaceFrac = state.dataHeatBal->space(spaceNum).FloorArea / zoneArea;
+                        } else {
+                            ShowSevereError(
+                                state,
+                                format("{}Zone floor area is zero when allocating ZoneBaseboard:OutdoorTemperatureControlled loads to Spaces.",
+                                       RoutineName));
+                            ShowContinueError(state,
+                                              format("Occurs for ZoneBaseboard:OutdoorTemperatureControlled object ={} in Zone={}",
+                                                     thisBBHeatInput.Name,
+                                                     state.dataHeatBal->Zone(zoneNum).Name));
+                            ErrorsFound = true;
+                        }
+                    }
+                    thisZoneBBHeat.CapatLowTemperature = IHGNumbers(1) * spaceFrac;
                     thisZoneBBHeat.LowTemperature = IHGNumbers(2);
-                    thisZoneBBHeat.CapatHighTemperature = IHGNumbers(3);
+                    thisZoneBBHeat.CapatHighTemperature = IHGNumbers(3) * spaceFrac;
```

Comment:
This was incorrect before when ZoneBaseboard:OutdoorTemperatureControlled was applied to a zone with more than one space, before it would duplicate the full power in each space. Now it's proportioned by floor area (maybe not the best proxy, but better than no proportion). ZoneBaseboard:OutdoorTemperatureControlled isn't used much (if at all) anyway.